homeassistant/components/insteon_local.py

```diff
@@ -71,6 +80,12 @@ def setup(hass, config):
             _LOGGER.error("Error on insteon_local hub check", exc_info=True)
         return False
 
+    linked = insteonhub.get_linked()
+
     hass.data['insteon_local'] = insteonhub
 
+    for insteon_platform in INSTEON_PLATFORMS:
+        _LOGGER.info("Load platform " + insteon_platform)
+        load_platform(hass, insteon_platform, DOMAIN, {'linked': linked})
```

Comment:
@wardcraigj I may need your help here to figure out the best way to do this.

Reply:
Ah, I see. @camrun91, so instead of passing `{'linked': linked}` into load_platform, and then assigning the linked variable like this `linked = discovery_info['linked'']` in the light, switch and fan components, remove line 83 in this file and also {'linked': linked} from line 89
and in light, switch and fan wherever you see `linked = discovery_info['linked'']` instead do `linked = insteonhub`